### ISSUE TITLE
Https error with the site

### ISSUE THREAD

**Issue by Kedstar99** [davearonson/pull-request-roulette#43]
![image](https://user-images.githubusercontent.com/8430304/66352190-0661a580-e957-11e9-8429-6396c1969073.png)
SSL certificate error on Firefox with the site. Mostly because the certificate seems to be only valid for herokuapp domains.

**Comment by davearonson** [comment]
Good catch!  A few years ago I would have quoted the Vaudeville doctor and said "So don't do that!" -- but nowadays HTTPS is pretty much the expectation.  If I remember rightly, there's some way to kluge it in for free through Cloudflare or something like that.  I'll look into it, and keep this open in the meantime.
